rllib/execution/buffers/replay_buffer.py

```diff
@@ -140,8 +141,9 @@ def sample(self, num_items: int) -> SampleBatchType:
             random.randint(0,
                            len(self._storage) - 1) for _ in range(num_items)
         ]
-        self._num_sampled += num_items
-        return self._encode_sample(idxes)
+        sample = self._encode_sample(idxes)
```

Comment:
I am guessing this is necessary if current RB holds fewer than num_items samples right?
so we may not return exactly num_items samples?
if that is the case, maybe:

```
self._num_sampled += len(idxes)
return self._encode_sample(idxes)
```

is a smaller fix?

Reply:
No, it's for the case that one item has more than 1 timesteps (sequences for e.g. R2D2).
So: num_items != num-timesteps (not always)

Reply:
ah!! ok. can you help add a comment about this here?
thanks.

Reply:
done